### DISCUSSION TITLE
Firmware update failed – manual FW restore – Emergency IFS reboot

### DISCUSSION THREAD

**Discussion by Ireinia** [Mr-MIBonk/M.I.B._More-Incredible-Bash#554] [General]
Hi Everyone , hoping someone can help me with the issue I have on my MHI2 H28.28.210_HIGH2_EU. I attempted a firmware update on my unit using bad SD card. The update has failed and now the unit is in the boot loop. I am attempting to perform manual FW restore as per https://mibwiki.one/doc/manual-firmware-restore-NglgI1RK9n however I ran into an issue with the Emergency IFS. After two minutes of working the unit reboots , and there is nothing I can do to stop that. slay -9 MIBEmergency does not stop the reboot. Any thoughts suggestions would be greatly appreciated.

**Comment by vonxbailey** [comment]
Did you ever get this fixed?

**Reply by Ireinia** [reply]
I got through the constant rebooting issue initially, After trying a few things ended up overwriting  Emergency IFS and bricking the unit. Bought a second hand MIB unit to replace.

**Reply by Ireinia** [reply]
Can try this command to stop the boot and reboot into Emergency IFS

Fn+B then any key 500ms
boot -t emerg
slay -9 MIBEmergency

**Reply by vonxbailey** [reply]
Do you mind providing the steps ? Doesn't have to be lengthy haha thanks in advance

**Reply by Ireinia** [reply]
What are you trying to do ? what's the issue ?

**Reply by vonxbailey** [reply]
The same exact problem you were having, but with my Galaxy Tab A7

**Reply by Ireinia** [reply]
Mate , don't know anything about mobile and how to get it fixed. The issue I had was with MIB (car infotainment system ) unit. Good luck
